split out wrapper scripts from debinstall.sh and add README (#82)
kensanata/mastodon-backup: contrib/README.md

# Contrib
This directory contains user-provided contributions intended to enhance
mastodon-backup.

## `upgrade_python-mastodon.sh` (by Izzy)
A script intended to upgrade outdated versions of `Mastodon.py` when installed
via your Linux distribution's package manager. As of this writing,

> mastodon-backup requires at least v1.5.1 of it for full functionality. It will

Reviewer: ```suggestion
mastodon-backup requires at least v1.5.1 for full functionality. It will
```

> work with v1.5.0 but then lack sone features (e.g. bookmark operations are not

Reviewer: ```suggestion
work with v1.5.0 with reduced functionality (e.g. bookmark operations are not
```
